doc/reference/reference_lua/box_info/synchro.rst

```diff
@@ -43,102 +46,126 @@ box.info.synchro
             Until the request is complete, any other incoming synchronous transactions and system requests
             will be delayed.
 
+        -   ``age`` (since version :doc:`3.2.0 </release/3.2.0>`) -- the time in seconds that the oldest entry currently
+            present in the queue has spent waiting for the quorum to collect.
+
+        -   ``confirm_lag`` (since version :doc:`3.2.0 </release/3.2.0>`) -- the time in seconds that the latest successfully
+            confirmed entry waited for the quorum to collect.
+
     *   ``quorum`` -- the resulting value of the
-        :ref:`replication_synchro_quorum <cfg_replication-replication_synchro_quorum>` configuration option.
+        :ref:`replication.synchro_quorum <configuration_reference_replication_synchro_quorum>` configuration option.
         Since version :doc:`2.5.3 </release/2.5.3>`, the option can be set as a dynamic formula.
         In this case, the value of the ``quorum`` member depends on the current number of replicas.
 
     **Example 1:**
 
-    In this example, the ``quorum`` field is equal to ``1``.
+    In this example, the ``quorum`` field is equal to `1`.
     That is, synchronous transactions work like asynchronous ones.
     `1` means that a successful WAL writing to the master is enough to commit.
 
     ..  code-block:: tarantoolsession
 
-        tarantool> box.info.synchro
+        box_info_synchro:instance001> box.info.synchro
         ---
         - queue:
             owner: 1
+            confirm_lag: 0
             term: 2
+            age: 0
             len: 0
             busy: false
           quorum: 1
         ...
 
     **Example 2:**
 
-    First, set a quorum number and a timeout for synchronous replication using the following command:
+    Example on GitHub: `box_info_synchro <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/replication/instances.enabled/box_info_synchro>`_
+
+    In this example, there are two instances:
+
+    - ``instance001`` is going to be the leader.
+    - ``instance002`` is a follower instance.
+
+    ..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/box_info_synchro/config.yaml
+        :language: yaml
+        :start-at: groups
+        :end-at: 3302
+        :dedent:
+
+    On the **first** instance, grant the user with the ``super`` role:
 
     ..  code-block:: tarantoolsession
 
-        tarantool> box.cfg{
-                 > replication_synchro_quorum=2,
-                 > replication_synchro_timeout=1000
-                 > }
+        box_info_synchro:instance001> box.schema.user.grant('guest', 'super')
 
-    Next, check the current state of synchronous replication:
+    After that, use ``box.ctl.promote()`` function to claim the queue:
 
     ..  code-block:: tarantoolsession
 
-        tarantool> box.info.synchro
+        box_info_synchro:instance001> box.ctl.promote()
+
+    Create a space called ``sync`` and enable synchronous replication on this space:
```

Comment:
```suggestion
    Create a space named ``sync`` and enable synchronous replication on this space:
```

Reply:
`call` is an overloaded term in our sphere (to call a method, e.g.), so I'd suggest a more specific word here